### PR TITLE
openflow_acl_test.xxx.telnet: Update main-vm name and passwd in cfg file

### PR DESCRIPTION
**Pull request by xiywang**
id: 1466686, 1467825

Signed-off-by: Xiyue Wang <xiywang@redhat.com>

### CODE REVIEW

**Comment by xiywang** [comment]
Rely on: [KVM-AUTOTEST] [PATCH] [PATCH]staf-kvm: add telnet server in RHEL7 kickstart

**Comment by xiywang** [comment]
python ConfigTest.py --testcase=openflow_acl_test.access_deny_all_target.telnet,openflow_acl_test.access_deny_special_target.telnet --guestname=RHEL.7.4 --platform=x86_64 --clone=no --netdst=ovs0
...
 (1/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_all_target.telnet: PASS (442.20 s)
 (2/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_special_target.telnet: PASS (325.87 s)

**Comment by xiywang** [comment]
@pezhang Please help to review. Thx

**Comment by pezhang** [comment]
Test 3 times, 2 of them are fail. Seems the results is not stable.
 
run1: 
(1/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_all_target.telnet: FAIL (378.23 s)
 (2/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_special_target.telnet: PASS (314.65 s)

Error info:
05:24:49 ERROR| Reproduced traceback from: /home/avocado-vt/avocado_vt/test.py:245
05:24:49 ERROR| Traceback (most recent call last):
05:24:49 ERROR|   File "/home/avocado-vt/avocado_vt/test.py", line 398, in _runTest
05:24:49 ERROR|     run_func(self, params, env)
05:24:49 ERROR|   File "/usr/lib/python2.7/site-packages/autotest/client/shared/error.py", line 140, in new_fn
05:24:49 ERROR|     return fn(*args, **kwargs)
05:24:49 ERROR|   File "/home/nic_hotplug/tp-qemu/qemu/tests/openflow_acl_test.py", line 412, in run
05:24:49 ERROR|     access_service(access_sys, access_targets, False, host_ip)
05:24:49 ERROR|   File "/home/nic_hotplug/tp-qemu/qemu/tests/openflow_acl_test.py", line 139, in access_service
05:24:49 ERROR|     raise error.TestFail(err_msg)
05:24:49 ERROR| TestFail: Still can not access virt-tests-vm2 after enable the accessCommand: telnet. Output: Failed to login avocado-vt-vm1 from virt-tests-vm2, err: Trying 10.73.75.242...
05:24:49 ERROR| Connected to 10.73.75.242.
05:24:49 ERROR| Escape character is '^]'.
05:24:49 ERROR| Connection closed by foreign host.
05:24:49 ERROR|     [context: Try to acess target to exam the enable rules]
05:24:49 ERROR| 
05:24:49 ERROR| 
05:24:49 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_framework-52.0-py2.7.egg/avocado/core/test.py:596
05:24:49 ERROR| Traceback (most recent call last):
05:24:49 ERROR|   File "/home/avocado-vt/avocado_vt/test.py", line 270, in runTest
05:24:49 ERROR|     self.fail(self.__status)
05:24:49 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_framework-52.0-py2.7.egg/avocado/core/test.py", line 762, in fail
05:24:49 ERROR|     raise exceptions.TestFail(message)
05:24:49 ERROR| TestFail: Still can not access virt-tests-vm2 after enable the accessCommand: telnet. Output: Failed to login avocado-vt-vm1 from virt-tests-vm2, err: Trying 10.73.75.242...
05:24:49 ERROR| Connected to 10.73.75.242.
05:24:49 ERROR| Escape character is '^]'.
05:24:49 ERROR| Connection closed by foreign host.
05:24:49 ERROR|     [context: Try to acess target to exam the enable rules]
05:24:49 ERROR| 


run 2:
 (1/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_all_target.telnet: PASS (439.54 s)
 (2/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_special_target.telnet: CANCEL (63.87 s)

Error info:
05:41:00 ERROR| Reproduced traceback from: /home/avocado-vt/avocado_vt/test.py:245
05:41:00 ERROR| Traceback (most recent call last):
05:41:00 ERROR|   File "/home/avocado-vt/avocado_vt/test.py", line 398, in _runTest
05:41:00 ERROR|     run_func(self, params, env)
05:41:00 ERROR|   File "/usr/lib/python2.7/site-packages/autotest/client/shared/error.py", line 140, in new_fn
05:41:00 ERROR|     return fn(*args, **kwargs)
05:41:00 ERROR|   File "/home/nic_hotplug/tp-qemu/qemu/tests/openflow_acl_test.py", line 384, in run
05:41:00 ERROR|     access_service(access_sys, access_targets, False, host_ip, ref=True)
05:41:00 ERROR|   File "/home/nic_hotplug/tp-qemu/qemu/tests/openflow_acl_test.py", line 138, in access_service
05:41:00 ERROR|     raise error.TestNAError(err_msg)
05:41:00 ERROR| TestNAError: Can not access localhost at the beginning. Please check your setup.Command: telnet. Output: Failed to login localhost from avocado-vt-vm1, err: 
05:41:00 ERROR| 
05:41:00 ERROR| Connection closed by foreign host.
05:41:00 ERROR|     [context: Try to access target before setup the rules]
05:41:00 ERROR| 
05:41:00 ERROR| 
05:41:00 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_framework-52.0-py2.7.egg/avocado/core/test.py:593
05:41:00 ERROR| Traceback (most recent call last):
05:41:00 ERROR|   File "/home/avocado-vt/avocado_vt/test.py", line 266, in runTest
05:41:00 ERROR|     self.cancel(self.__status)
05:41:00 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_framework-52.0-py2.7.egg/avocado/core/test.py", line 806, in cancel
05:41:00 ERROR|     raise exceptions.TestCancel(message)
05:41:00 ERROR| TestCancel: Can not access localhost at the beginning. Please check your setup.Command: telnet. Output: Failed to login localhost from avocado-vt-vm1, err: 
05:41:00 ERROR| 
05:41:00 ERROR| Connection closed by foreign host.
05:41:00 ERROR|     [context: Try to access target before setup the rules]
05:41:00 ERROR| 
05:41:00 ERROR| CANCEL 2-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_special_target.telnet -> TestCancel: Can not access localhost at the beginning. Please check your setup.Command: telnet. Output: Failed to login localhost from avocado-vt-vm1, err:


run 3:
 (1/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_all_target.telnet: PASS (442.58 s)
 (2/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.openflow_acl_test.access_deny_special_target.telnet: PASS (329.02 s)

File: qemu/tests/cfg/openflow_acl_test.cfg [hunk 1]
```diff
@@ -42,7 +42,7 @@
             access_cmd = ssh
             target_port = 22
             host_username = root
-            host_password = redhat
+            host_password = kvmautotest
```
Comment (suqinhuang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: pls sent it internal

Comment (xiywang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: what do you mean by "sent it internal"? should i put the password in staf?

**Comment by xiywang** [comment]
@pezhang @suqinhuang Updated, please help to review again. Thank you.

**Comment by pezhang** [comment]
(1/4) smp_4.1024m.repeat1.pre_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_native: PASS (1607.90 s)
 (2/4) smp_4.1024m.repeat1.pre_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rh_kernel_update: PASS (962.90 s)
 (3/4) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.openflow_acl_test.access_deny_all_target.telnet: PASS (506.98 s)
 (4/4) smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.openflow_acl_test.access_deny_special_target.telnet: PASS (306.23 s)


Looks good to me.
Acked-by: Pei Zhang pezhang@redhat.com

**Comment by xiywang** [comment]
@luckyh Please help to review, thanks.

File: qemu/tests/cfg/openflow_acl_test.cfg [hunk 2]
```diff
@@ -79,7 +77,7 @@
                 access_cmd_linux = curl -L www.redhat.com --connect-timeout 10
                 access_cmd_windows = cd C:\lynx && lynx.exe --dump www.redhat.com
             ssh, telnet:
-                vms += " virt-tests-vm2"
+                vms = "virt-tests-vm1 virt-tests-vm2"
```
Comment (luckyh) [inline qemu/tests/cfg/openflow_acl_test.cfg]: Where used this name?

File: qemu/tests/cfg/openflow_acl_test.cfg [hunk 3]
```diff
@@ -42,7 +42,6 @@
             access_cmd = ssh
             target_port = 22
             host_username = root
-            host_password = redhat
```
Comment (suqinhuang) [inline qemu/tests/cfg/openflow_acl_test.cfg]: I think it's ok to keep this params, user would update it according to their env

Comment (xiywang) [reply qemu/tests/cfg/openflow_acl_test.cfg]: Users may probably forget or don't know they need to modify this parameter before they do the test. Then they'll get test fail. I don't agree this is a good way to users.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**